andela/suyabay
# Ch other user interface fixes 121399895

#### What does this PR do?

This PR performs some more fixes to the user interface.
#### Description of Task to be completed?
- Change the `Channels` on the sidebar to `View all Channels`.
- Also remove the `11` badge on this menu.
- Include the number of views that an episode has.
- Once an episode is viewed, increment the number of views of that episode.
- Sort the episodes by the number of views.
#### How should this be manually tested?

> Visit the page for all episodes.
> Notice that all the episodes are sorted by their number of views, in descending order.
> Also note the number of views of a particular episode and click to visit its page.
> Notice that the number of views of the episode increases by 1.
#### Any background context you want to provide?

A user should be able to know the number of views of a particular episode.
#### What are the relevant pivotal tracker stories?

The related PT story can be viewed [here](https://www.pivotaltracker.com/story/show/121399895).
#### Screenshots (if appropriate)

![sortviews](https://cloud.githubusercontent.com/assets/17028608/16016114/0fed0058-3191-11e6-9420-28dde94cedf8.png)
![incrementviews](https://cloud.githubusercontent.com/assets/17028608/16016113/0fec12b0-3191-11e6-875b-f95320e7f26d.png)
#### Questions:

There are no questions at this point.

[@unicodeveloper](https://github.com/unicodeveloper)
